packages/babel-plugin-transform-object-rest-spread/test/fixtures/object-rest/for-x-array-pattern/output.js

```diff
@@ -1,45 +1,65 @@
 const _excluded = ["a"],
   _excluded2 = ["a"],
-  _excluded3 = ["a"];
+  _excluded3 = ["a"],
+  _excluded4 = ["a"];
 // ForXStatement
-for (const _ref of []) {
-  const [_ref2] = _ref;
-  const {
-      a
-    } = _ref2,
-    b = babelHelpers.objectWithoutProperties(_ref2, _excluded);
-}
-for (var _ref3 of []) {
-  [_ref4] = _ref3;
```

Comment:
It seems we didn't declare `_ref4`? This is also the case in the new output.

Reply:
Fixed in https://github.com/babel/babel/pull/17281/commits/7088af9e61b7c4bf47c39b17df0c558bef78cdc6